### DISCUSSION TITLE
How to make a favorite for Star Trek movies

### DISCUSSION THREAD

**Discussion by CptKirk-1701** [OpenSageTV/Community#125] [General]
Is there a way to create a favorite that will record only the Star Trek movies, but not all the different tv shows?

**Comment by wayner9** [comment]
Good question.  When you do an EPG search in the web UI you can select a search string of star trek a category of Movies and that seems to work.  But I don't think you can do that in the Favorites Manager.  I wasn't able to 100% test this as I don't have any Star Trek movies in my guide but I was able to do this for Spider Man as there are both TV shows and movies for that.

But to be honest, aren't there just a finite number of Star Trek movies?  Can't you just acquire these by "other means" and add them to your library in SageTV or Plex or whatever?

**Reply by trk2** [reply]
You can do this as a favorite.  Create a new favorite for a category of Movies.  Then edit the newly created favorite and add a key word of "Star Trek".

**Comment by CptKirk-1701** [comment]
Thanks.  I figured I would just slowly get them when they were broadcast.  But I decided to get the first 10 (the old ones) as a boxed Blueray set for $40.

:)
